runtime/j9vm/java9vmi.c

```diff
@@ -989,6 +990,18 @@ JVM_DefineModule(JNIEnv * env, jobject module, jstring version, jstring location
 							success = vmFuncs->setBootLoaderModulePatchPaths(vm, j9mod, (const char *)nameUTF);
 							if (FALSE == success) {
 								goto nativeOOM;
+							} else {
+								const char* moduleName = "openj9.sharedclasses";
+
+								if (0 == strncmp(nameUTF, moduleName, 20)) {
+									J9VMDllLoadInfo *entry = FIND_DLL_TABLE_ENTRY(J9_SHARED_DLL_NAME);
+
+									if ((NULL != entry)
+										&& (J9_ARE_ALL_BITS_SET(entry->loadFlags, FAILED_TO_LOAD))
```

Comment:
Could the J9_SHARED_DLL_NAME ever fail to be found? I'm wondering the message should also be printed when (NULL == entry).

Reply:
Right, it is better to also print this message when entry is null.